pkg/toolset/network_bootstrap.go

```diff
@@ -31,33 +31,30 @@ import (
 
 // CoordinatorState is the JSON representation of a coordinator state.
 type CoordinatorState struct {
-	LatestMilestoneIndex   uint32 `json:"latestMilestoneIndex"`
-	LatestMilestoneBlockID string `json:"latestMilestoneBlockID"`
-	LatestMilestoneID      string `json:"latestMilestoneID"`
-	LatestMilestoneTime    int64  `json:"latestMilestoneTime"`
+	LatestMilestoneIndex   iotago.MilestoneIndex `json:"latestMilestoneIndex"`
+	LatestMilestoneBlockID string                `json:"latestMilestoneBlockID"`
+	LatestMilestoneID      string                `json:"latestMilestoneID"`
+	LatestMilestoneTime    int64                 `json:"latestMilestoneTime"`
```

Comment:
we should probably switch to uint32 here and in inx-coordinator too

Reply:
But in the COO state we used higher resolution (nanoseconds) in the state for a reason.
(I don't remember why :kek:)